apache/rocketmq/v2/service.proto

```diff
@@ -248,6 +248,8 @@ message Settings {
 
   optional Endpoints access_point = 2;
 
+  optional string deployment_name = 3;
```

Comment:
1. Is there a way to tag this field is of [out] type, up to servers to fill?
2. rename deployment_name --> deployment